src/tools/cephfs/top/cephfs-top

```diff
@@ -42,6 +42,9 @@ FS_TOP_FS_SELECTED_APP = 'SELECTED_FS_APP'
 # version match b/w fstop and stats emitted by mgr/stats
 FS_TOP_SUPPORTED_VER = 2
 
+# python version which this code needs
+FS_TOP_SUPPORTED_PYTHON_VER = 3.6
```

Comment:
Shouldn't this be a part of CMake checks which complain when the python version mismatches?

Reply:
Do you mean in `do_cmake.sh` during build ?

Reply:
In CMakeLists.txt, where you'd have access to `MGR_PYTHON_VERSION_MAJOR` and `MGR_PYTHON_VERSION_MINOR` variables.

Reply:
Done the check.

Reply:
Sure. Do we really need this after the cmake check?